app/assets/stylesheets/components/_buttons.scss

```diff
@@ -0,0 +1,17 @@
+.btn-gold {
+  background-color: transparent;
+  padding: 10px 20px;
+  text-decoration: none;
+  font-size: 16px;
+  border-radius: 4px;
+  display: inline-block;
+  text-align: center;
```

Comment:
The text is already centered because your button is nothing but the text and the padding. You're not giving the button a specific width and as such it's only taking the required width to contain the text inside of it.
Because you don't need the text-center you don't need the display inline block either.